compat_test.go

```diff
@@ -534,35 +558,27 @@ var recursiveThirdPartyCaveatMacaroons = []macaroonSpec{{
 
 func (*suite) TestVerify(c *gc.C) {
 	for i, test := range verifyTests {
-		c.Logf("test %d: %s", i, test.about)
-
 		for _, impl := range mcompat.Implementations {
-			c.Logf("- implementation: %s", impl.Name)
+			c.Logf("\nimplementation %s", impl.Name)
```

Comment:
The \n at the start is delberate here - it adds an extra piece of vertical white space so that the output is in more obvious sections. A \n at the end would be ignore (and I want "implementation" to start the paragraph not end it)